airflow/www/templates/airflow/dags.html

```diff
@@ -39,7 +39,7 @@
   <meta name="last_dag_runs_url" content="{{ url_for('Airflow.last_dagruns') }}">
   <meta name="dag_stats_url" content="{{ url_for('Airflow.dag_stats') }}">
   <meta name="task_stats_url" content="{{ url_for('Airflow.task_stats') }}">
-  <meta name="tree_url" content="{{ url_for('Airflow.tree') }}">
+  <meta name="grid_url" content="{{ url_for('Airflow.grid', dag_id='${dag_id}') }}">
```

Comment:
Why do we refer to `${dag_id}` here and `$dag_id` in. `dags.js`? Is that a typo?

Reply:
Whoops yes typo!

Reply:
Fixed to `__DAG_ID__~ as `$` and `{` are both URL escaped so those characters made it more complex to handle